Dockerfile

```diff
@@ -7,6 +7,10 @@ RUN apk add g++ make python3
 RUN apk add --update --upgrade busybox libretls openssl zlib curl
 
 COPY . /app
+
+# Add this the service Dockerfile, at the final stage if multi-stage
```

Comment:
Hi from this project we can tell this file is not being used during deployment to ddev/stg/prod
https://bitbucket.org/atlassian/github-for-jira-deployment/src/master/

It should the `Dockerfile.prod`

Reply:
@gxueatlassian do we have a separate dockerfile for stg and ddev?

Reply:
I think `ddev` `stg` and `prod` shared the same docker file, but you can adjust that in the deployment scripts above.